File: Bank.py
class Bank:
    def __init__(self):
        self.employeeList = []

    def list_employees(self):
        for employee in self.employeeList:
            print(" * {}".format(employee.get_user_name()))

    def get_user(self):
        return self.employeeList[0]

    def check_user(self, emp):
        for employee in self.employeeList:
            if emp == employee.get_user_name():
                return True
        return False

    def check_account_id(self, emp, id):
        if id < 0 or id >= len(emp.accountList):
            return False
        else:
            return True

    def store_user(self, user):
        self.employeeList.append(user)

File: Bank.java
import java.util.*;
public class Bank {
    public static ArrayList<Employee> employeeList = new ArrayList<>();

    public void listEmployees() {
        for (int i = 0; i < employeeList.size(); i++) {
            System.out.println(" * "+employeeList.get(i).getUserName());
        }
    }

    public Employee getUser(){
        return employeeList.get(0);
    }

    public boolean checkUser(String emp){
        boolean found = false;
        for (int i = 0; i < employeeList.size(); i++) {
            if(emp.equals(employeeList.get(i).getUserName())){
                found = true;
                break;
            }
        }
        return found;
    }

    public boolean checkAccountID(Employee emp, int id){
        boolean found = false;
        if(id < 0 || id > emp.getAccountList().size()){
            found = false;
        }else{
            found = true;
        }
        return found;
    }

    public void storeUser(Employee user) {
        employeeList.add(user);
    }

    public ArrayList<Employee> getEmployeeList() {
        return employeeList;
    }

}
